system/Filters/PageCache.php

```diff
@@ -65,6 +66,10 @@ public function after(RequestInterface $request, ResponseInterface $response, $a
     {
         assert($request instanceof CLIRequest || $request instanceof IncomingRequest);
 
+        if ($this->isNotCacheableMethod($request)) {
```

Comment:
So what's cachable? GET and HEAD? If that's all, and HEAD isn't something we really need to worry about caching, wouldn't it be simpler to just check if `$request->getMethod() !== 'get'`?

Reply:
ResponseCache also caches the headers, so in theory, it can be useful for HEAD... however, as I mentioned in my comment, I would make it configurable and stick to the GET by default.

Reply:
> So what's cachable?

GET, HEAD, POST, PATCH, CLI. 
But a dev must put `$this->cachePage($n);` in controller.

Reply:
> But a dev must put $this->cachePage($n); in controller.

Ok, that changes a lot.

Reply:
I dropped the check.

To begin with, page caching must be specified by developers in controllers. 
There is no need for the filter to dare to check the HTTP method to limit the cache functionality.

Besides, if a developer wants to determine to enable caching by HTTP method, they can customize the filter.